.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.4.2
```

Comment:
Has this hot fix been applied to the master branch yet?

Reply:
Nope, it would come in as part of this PR. Why do you ask?